nixops/backends/__init__.py

```diff
@@ -247,16 +247,29 @@ def reboot_sync(self, hard: bool = False) -> None:
         """Reboot this machine and wait until it's up again."""
         self.reboot(hard=hard)
         self.log_start("waiting for the machine to finish rebooting...")
-        nixops.util.wait_for_tcp_port(
```

Comment:
Should we explicitly add a deprecation message to this, or even delete the function?